pom.xml

```diff
@@ -113,6 +113,16 @@
         </dependency>
 
         <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-multibranch</artifactId>
+            <version>2.9</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>github-branch-source</artifactId>
```

Comment:
github-oauth duplicated class to not depend on github-plugin, now it hard depend on github-branch-source.

Reply:
https://github.com/jenkinsci/github-oauth-plugin/blob/master/src/main/java/org/jenkinsci/plugins/GitHubRepositoryName.java#L34-L39